source/index.html

```diff
@@ -1,12 +1,229 @@
+                <p class="quote__text">Я молодая мама, но всё равно фотографии не набирали и близко такого количества
+                  лайков, как у популярных инстамамочек. В отчаянии, я накупила аксессуаров и игрушек в Мишке, и мои
+                  фотографии сразу стали более стильными, а также набирают больше лайков!</p>
+                <cite class="quote__author-name">Анастасия Красильникова</cite>
+                <address>@misssssiskras</address>
+              </div>
+            </blockquote>
+          </li>
+          <li class="slider__item">
+            <blockquote class="slider__quote quote">
+              <div class="quote__author-info">
+                <p class="quote__text">Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium
+                  doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi
+                  architecto beatae vitae dicta sunt explicabo. Nemo enim ipsam voluptatem quia voluptas sit aspernatur
+                  aut odit aut fugit, sed quia consequuntur magni dolores eos qui ratione voluptatem sequi nesciunt.</p>
+                <cite class="quote__author-name">Татьяна Шапошникова</cite>
+                <address>@lorryipsum</address>
+              </div>
+            </blockquote>
+          </li>
+          <li class="slider__item">
+            <blockquote class="slider__quote quote">
+              <div class="quote__author-info">
+                <p class="quote__text">Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium
+                  doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi
+                  architecto beatae vitae dicta sunt explicabo.</p>
+                <cite class="quote__author-name">Светлана Кольцова</cite>
+                <address>@lorrrrrrryipsum</address>
+              </div>
+            </blockquote>
+          </li>
+        </ul>
+        <div class="reviews__button button-slide">
+          <button class="button-slide__prev" type="button">
+            <span class="visually-hidden">Предыдущий отзыв</span>
+          </button>
+          <button class="button-slide__next" type="button">
+            <span class="visually-hidden">Следующий отзыв</span>
+          </button>
+        </div>
+        <a href="#" class="reviews__button">Написать</a>
+      </div>
+    </section>
+    <section class="contacts">
+      <h2 class="contacts__title">Контакты</h2>
+      <ul class="contacts__list">
+        <li class="contacts__item email">
+          <span class="email__title">e-mail: </span>
+          <a class="email__subtitle" href="mailto:info@mimimishkashop.ru">info@mimimishkashop.ru</a>
+        </li>
+        <li class="contacts__item address">
+          <span class="address__title">адрес: </span>
+          <p class="address__subtitle">Санкт-Петербургул. Большая Конюшенная, д. 19/8, офис 101</p>
```

Comment:
Сделай лучше сразу так
```suggestion
          <p class="address__subtitle">
            Санкт-Петербург<br>
            ул. Большая Конюшенная<br>
            д. 19/8, офис 101
          </p>
```